tests/test_plot.py

```diff
@@ -230,6 +230,12 @@ def test_kde_plot_1d(plot_1d):
         plot_1d(ax, data, q=0.9)
         plot_1d(ax, data, q=(0.1, 0.9))
 
+        # Check iso-probability code
+        plot_1d(
+            ax, data, filled1d=True)
+        plot_1d(
+            ax, data, filled1d=True, levels=[0.2, 0.6, 0.8])
```

Comment:
Do we need the linebreaks here?

Reply:
No, this is not needed. I will update it in the next commit.